.github/workflows/analyze.yml

```diff
@@ -8,9 +8,17 @@ jobs:
               uses: shivammathur/setup-php@v2
               with:
                   php-version: 8.2
+            - name: Install Node
+              uses: actions/setup-node@v3
+              with:
+                  node-version: lts/iron
             - name: Checkout code
               uses: actions/checkout@v3
             - name: Install PHP dependencies
               run: composer install --no-interaction --no-progress --no-scripts
+            - name: Remove private NPM package
+              run: npm uninstall closer-look --save
+            - name: Install Javascript dependencies
+              run: npm ci
```

Comment:
Why does the action need the js packages installed to analyze php?

Reply:
Because we specify to explude `node_modules`, and the directory is not found unless it's installed. Rather than configuring different implementations for PHPStan, I thought it better to install the NPM packages to have a uniform environment.